### PR TITLE
New WORKING clone

### PR DESCRIPTION
**Pull request by clawgrip**
------------------------------
Sky Shark (bootleg) [jordigahan]

### CODE REVIEW

File: src/mame/mame.lst [hunk 1]
```diff
@@ -41478,6 +41478,7 @@ gulfwar2a                       //
 hishouza                        // B02 / TP-007 (c) 1987 Taito Corporation (Japan)
 hishouzab                       // bootleg
 ktiger                          // B30 / TP-011 (c) 1987 Taito Corporation (Japan)
+skysharbl                       // bootleg
 skyshark                        // B02 / TP-007 (c) 1987 Taito America Corporation + Romstar license (US)
 skysharka                       // B02 / TP-007 (c) 1987 Taito America Corporation + Romstar license (US)
```
Comment (cuavas) [inline src/mame/mame.lst:41483]: This isn’t in order.

File: src/mame/drivers/twincobr.cpp [hunk 2]
```diff
@@ -1236,46 +1236,97 @@ ROM_START( fnshark ) // based on a different version of the game code? (only a ~
...
+	ROM_LOAD( "1e.ic5", 0x04000, 0x04000, CRC(543bbb81) SHA1(8aabc3d4f14b6531af000008b50cf09bb6cd003f) ) // 1xxxxxxxxxxxxx = 0xFF
+	ROM_LOAD( "1f.ic6", 0x08000, 0x04000, CRC(d357f494) SHA1(e83560e551ea15cac903e9a8c5ce57d51af175c7) ) // 1xxxxxxxxxxxxx = 0xFF
+
+	// Same data on larger EPROMs with first half empty
+//  ROM_LOAD( "5.bin", 0x08000, 0x04000, NO_DUMP ) // Not dumped on this set
+//  ROM_CONTINUE(0x08000,0x04000)
+//  ROM_LOAD( "6.bin", 0x04000, 0x04000, NO_DUMP ) // Not dumped on this set
+//  ROM_CONTINUE(0x04000,0x04000)
+//  ROM_LOAD( "7.bin", 0x00000, 0x04000, NO_DUMP ) // Not dumped on this set
+//  ROM_CONTINUE(0x00000,0x04000)
```
Comment (cuavas) [inline src/mame/drivers/twincobr.cpp:1288]: So what’s going on here?  Do these ROMs actually exist undumped?  If they are, I’m not sure it’s helpful to add the game in the current state.  There seem to be an increasing number of games added where we just hope certain ROMs are the same as other sets.  On the other hand, if these ROMs don’t actually exist, the comment is unhelpful.

Comment (clawgrip) [reply src/mame/drivers/twincobr.cpp]: Upps! No. I was thinking on them when Jordi told me that he left four (four, not three) ROMs undumped, but those are for the MCU code, not GFX.
I've removed the comment and add a note about the undumped MCU code.

File: src/mame/mame.lst (hunk 1 above)
Comment (clawgrip) [reply src/mame/mame.lst:41483]: Sorry about the question, but... It isn't?

Comment (happppp) [reply src/mame/mame.lst:41483]: If you didn't remove the "k" then I suppose ;)

I'd keep the "k". By the way, IMO the setnames for the fshark bootlegs in this driver are a mess:
fsharkb, fsharkbt, fnshark

**Comment by Quench0** [comment]
Until the four TMS32010 ROMs are dumped, can you add a comment on what type of bipolar PROMs they are?

**Comment by clawgrip** [comment]
> Until the four TMS32010 ROMs are dumped, can you add a comment on what type of bipolar PROMs they are?

Done!

File: src/mame/mame.lst (hunk 1 above)
Comment (clawgrip) [reply src/mame/mame.lst]: I've renamed the set keeping the "k".

Thx!

**Comment by Tafoid** [comment]
A couple of issues have been observed, noted here:
https://mametesters.org/view.php?id=8021

1q.1ic7 - possible typo? should be 1q.ic17?
1l.ic12 - Listed twice? 2 separate crc/sha1 listed?

Probably a combination of mistypes and copy/paste hangovers.  Can you please fix this?  I'd rather not partially fix since you have access to original PCB documents/pictures and the like.

**Comment by clawgrip** [comment]
Fixed on PR #8262

Thanks a lot

**Pull request merged**